datafusion/core/src/physical_plan/planner.rs

```diff
@@ -1746,8 +1746,6 @@ mod tests {
     async fn errors() -> Result<()> {
         let bool_expr = col("c1").eq(col("c1"));
         let cases = vec![
-            // utf8 < u32
-            col("c1").lt(col("c2")),
```

Comment:
Just curious why this was removed from the test?

Reply:
This test was specifically for illegal type coercions and this is no longer illegal